src/redist/targets/BundledManifests.targets

```
@@ -104,6 +104,22 @@
     
     <Copy SourceFiles="@(ManifestContent)"
           DestinationFolder="$(RedistLayoutPath)sdk-manifests/%(DestinationPath)"/>
-  
+
+    <PropertyGroup>
+      <WorkloadSetVersion Condition="'$(DotNetFinalVersionKind)' == 'release'">$(VersionPrefix)-baseline$(_BuildNumberLabels)</WorkloadSetVersion>
+      <WorkloadSetVersion Condition="'$(DotNetFinalVersionKind)' != 'release'">$(Version)</WorkloadSetVersion>
+      <RealFormattedManifestPaths>$(RedistLayoutPath)sdk-manifests\$(NetFeatureBand)\workloadsets\$(WorkloadSetVersion)</RealFormattedManifestPaths>
+    </PropertyGroup>
+
+    <ItemGroup>
+    </ItemGroup>
+
+    <ItemGroup>
+      <FormattedBaselineManifest Include="{" />
+      <FormattedBaselineManifest Include="&quot;%(BundledManifests.Identity)&quot;: &quot;%(BundledManifests.Version)/%(BundledManifests.FeatureBand)&quot;,"/>
+      <FormattedBaselineManifest Include="}" />
+    </ItemGroup>
+
+    <WriteLinestoFile File="$(RealFormattedManifestPaths)\Baseline.WorkloadSet.json" Lines="@(FormattedBaselineManifest)" />
```

The resolver looks for `*.workloadset.json`, so this needs to be lowercase so that it will work on case-sensitive file systems.

```suggestion
    <WriteLinestoFile File="$(RealFormattedManifestPaths)\baseline.workloadset.json" Lines="@(FormattedBaselineManifest)" />
```